### PR TITLE
Introducing Aya Guru on Gurubase.io

### PR DESCRIPTION
**Pull request by kursataktas**
Hello team,

I'm the maintainer of [Anteon](https://github.com/getanteon/anteon). We have created Gurubase.io with the mission of building a centralized, open-source tool-focused knowledge base. Essentially, each "guru" is equipped with custom knowledge to answer user questions based on collected data related to that tool.

I wanted to update you that I've manually added the [Aya Guru](https://gurubase.io/g/aya) to Gurubase. Aya Guru uses the data from this repo and data from the [docs](https://aya-rs.dev/) to answer questions by leveraging the LLM.

In this PR, I showcased the "Aya Guru", which highlights that Aya now has an AI assistant available to help users with their questions. Please let me know your thoughts on this contribution.

Additionally, if you want me to disable Aya Guru in Gurubase, just let me know that's totally fine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Crates.io][crates-badge]][crates-url]
 ![License][license-badge]
 [![Build status][build-badge]][build-url]
-[![Book][book-badge]][book-url]
+[![Book][book-badge]][book-url] [![Gurubase][gurubase-badge]][gurubase-url]
```
Comment (vadorovsky) [inline README.md:6]: Let's keep it in a separate line?

```suggestion
[![Book][book-badge]][book-url]
[![Gurubase][gurubase-badge]][gurubase-url]
```

**Review by vadorovsky — APPROVED** [review]
I'm personally fine with adding it and I can see that the results produced by Gurubase make much more sense than GPT 4o or Qwen2.5 for the same question asked directly. This is actually nice:

https://gurubase.io/g/aya/write-xdp-program-firewall-rules-aya

**Review by tamird — APPROVED** [review]
LGTM with a squash on merge.

**Comment by kursataktas** [comment]
Hi everyone,

I’d like to update you on the release of the Maintainer Panel feature on Gurubase. With this panel, you can add, remove, or update data sources, as well as change the logo and more. You can find the details [here](https://github.com/getanteon/gurubase?tab=readme-ov-file#how-to-claim-a-guru). 

In the near future, I’m planning to include analytics insights in this panel, such as the number of questions asked, the most frequently asked ones, and more. I’ll be sure to update this thread once it’s available. However, in case I miss it, I highly recommend joining our [Discord channel](https://discord.gg/9CMRSQPqx6) to stay updated.

If you’d prefer that I don’t update this thread anymore, please let me know.

**Comment by kursataktas** [comment]
Hello Everyone,

We recently released the embeddable "Ask AI" widget, which allows you to integrate your Guru’s Q&A capability into websites, documentation, and more.

The installation process is straightforward, as explained in [this GitHub repository](https://github.com/getanteon/gurubase-widget).

You can see it in action on the [Anteon](https://getanteon.com/) and [NetAlertX](https://netalertx.com/) websites.

**Comment by tamird** [comment]
@kursataktas in the aya discord we use a "forum" channel for user questions: https://discord.com/channels/855676609003651072/1291061384653832264

I can't seem to grant the discord bot access to this "forum", only to text channels.

**Comment by kursataktas** [comment]
@tamird I enabled Codebase indexing, and it has been successfully indexed. You can confirm this on the Guru Edit UI. I'm looking into the Discord "forum" channel issue, and I'll update this thread.

**Comment by kursataktas** [comment]
@tamird you should be able to add the bot to the 'forum' channel. Can you confirm?

**Comment by tamird** [comment]
@kursataktas I was able to add it, but sending a test message failed.

**Comment by kursataktas** [comment]
@tamird, you probably need to reconnect the bot to Discord, as it would require new permissions for the forum channel. I'm not entirely sure if this is the case, but based on the logs, this seems to be the main reason. Can you try reconnecting?

**Comment by tamird** [comment]
@kursataktas reconnected, same result.

**Comment by tamird** [comment]
Sending a test message to the general channel worked, though.

**Comment by kursataktas** [comment]
@tamird, I'll investigate further and get back to you once I find the root cause. It works on our 'forum' channel, and currently, I can't reproduce the issue. If you think it might be due to a custom configuration on your Discord channel, please let me know.

**Comment by kursataktas** [comment]
> @kursataktas might be this?
> 
> Screenshot 2025-02-14 at 15 32 41

Yes, that's the issue. It works as expected.

**Comment by tamird** [comment]
@kursataktas it'd be nice to improve the error message, or somehow allow the user to specify tags so that a self-test is possible. I expect you'll have other users with this problem.

**Pull request merged**